youtube_dl/extractor/infomaniak.py

```diff
@@ -0,0 +1,88 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    parse_duration,
+)
+
+
+class InfomaniakVod2IE(InfoExtractor):
+    IE_NAME = 'infomaniak:vod2'
+    _VALID_URL = r'https?://player\.vod2\.infomaniak\.com/embed/(?P<id>[0-9a-z]+)'
+    _TEST = {
+        'url': 'https://player.vod2.infomaniak.com/embed/1jhvl2uqg6ywp',
+        'info_dict': {
+            'id': '1jhvl2uqg6xis',
+            'display_id': '1jhvl2uqg6ywp',
+            'ext': 'mp4',
+            'title': 'Conférence à Dyo, octobre 2022',
+            'thumbnail': r're:^https?://.*\.(?:jpe?g|png)$',
```

Comment:
`re:` already implies a match anchored at the start, and the path extension surely can't be immediately after the `//`.
```suggestion
            'thumbnail': r're:https?://.+\.(?:jpe?g|png)$',
```

Reply:
Thanks! Updated the thumbnail regex to drop the redundant ^ and make it more realistic (no immediate // → extension). Now using: r're:https?://.+\.(?:jpe?g|png)$'.